packages/schematics/schematics/schematic/factory.ts

```diff
@@ -8,31 +8,36 @@
 import { Path, strings } from '@angular-devkit/core';
 import {
   Rule,
+  SchematicContext,
+  Tree,
   apply,
   mergeWith,
   move,
-  partitionApplyMerge,
-  template,
-  url,
+  partitionApplyMerge, template, url,
```

Comment:
Nit: can we keep these in a single line please? Thanks

Reply:
Yes. Updated.